language/transaction-builder/generator/src/common.rs

```diff
@@ -34,7 +34,7 @@ fn quote_type_as_format(type_tag: &TypeTag) -> Format {
             U8 => Format::Bytes,
             Vector(type_tag) => {
                 if type_tag.as_ref() == &U8 {
-                    Format::Seq(Box::new(Format::Bytes))
+                    Format::TypeName("VecBytes".into())
```

Comment:
I thought `VecBytes` would be only used internally by client code to provide convenient serialization methods for the anonymous type `Vec<Bytes>` but I suppose "forcing" it in the abi interfaces is fine as well.

I guess it depends if we think that many more anonymous types `Vec<Vec<u64>>`, `Vec<Option<T>>`, etc will be needed in the future. In this case, I would keep the natural translation in the abi interfaces and instead of defining `VecBytes` at all, I would perhaps just define generic helper functions somewhere, possibly in the serde-generate runtime(s) (e.g. to serialize `Vec<T>` provided a lambda to serialize `T`). Such helper functions would be less convenient to use but composable.